meson.build

```diff
@@ -0,0 +1,22 @@
+project('Hyprland', 'cpp', 'c',
+  version : '0.1',
```

Comment:
Should be `0.5.0`? Maybe `.beta` too?

Reply:
I will not update it, so unless it updates itself, this should stay different from the hyprland release vers

Reply:
If the `.git` directory is available inside Actions, we could make an action that auto-updates it, the way flake.lock is updated.

Reply:
```bash
#!/bin/sh

CRT_REV=$(git show-ref --tags --head --abbrev | head -n 1 | head -c 7)
TAG_REV=$(git show-ref --tags --abbrev | tail -n 1 | head -c 7)
CRT_VER=$(sed -nEe "/([0-9]+(\.[0-9a-zA-Z]+)+)/{p;q;}" meson.build | awk -F\' '{print $2}')
VERSION=$(git show-ref --tags --abbrev | tail -n 1 | tail -c +20)

if [[ $TAG_REV = $CRT_REV ]] || [[ $CRT_VER != $VERSION ]]; then
  sed -Ei "s/([0-9]+(\.[0-9a-zA-Z]+)+)/$VERSION/g" meson.build
fi
```
tested and it works. maybe the second check isn't necessary if the proper version is set in this PR.